### DISCUSSION TITLE
Add Latin Hypercube Simulation (LHS) functionality

### DISCUSSION THREAD

**Discussion by RyanConway91** [pypest/pyemu#423] [Ideas]
Future enhancement idea:

It would be cool if some of the classic PEST Latin Hypercube Simulation (LHS) functionality was available in pyemu.

**Comment by jtwhite79** [comment]
Hey Ryan - I think there are a few py based implementations floating around and we could wrap them easily enough...but I've been hesitant because my understanding is that LHS doesn't scale to the kinds of dimensions we deal with.  Is that outdated information?

**Reply by RyanConway91** [reply]
I wasn't aware of that limitation but I guess it makes sense. I have not tried LHS at a very high dimensionality yet. Some quick googling indicated that "Markov chain Monte Carlo" could be an alternative. I'll look into this a bit more. Thanks!
